.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,46 @@
+name: Local Env CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  local-env:
+    name: Setup Local Environment
+    runs-on: ubuntu-latest
```

Comment:
I recommend you to try to always fix docker images versions and never(extremely rarely) use latest tags.